addon/ajax-request.js

```diff
@@ -160,6 +160,15 @@ export default class AjaxRequest {
   }
 
   /**
+   * calls `request()` but forces `options.type` to `DELETE`
+   * alias for `del()`
+   * @public
+   */
+  delete(...args) {
+    return this.del(...args);
```

Comment:
@jaswilli exactly, that's what I meant.  Sorry that I didn't make that clear enough.